_opensearch/rest-api/ingest-apis/delete-ingest.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Delete a pipeline
+parent: Ingest APIs
+grand_parent: REST API reference
+nav_order: 14
+---
+
+# Delete a pipeline
+
+If you no longer want to use an ingest pipeline, use the delete ingest pipeline API operation.
+
+## Example
+
+```
+DELETE _ingest/pipeline/{id}
+```
+
```

Comment:
We want to use examples that customers can copy and paste, so consider using an actual ID and not {} here. 

Also, this page is missing a "Path and HTTP methods" section.